pyproject.toml

```diff
@@ -1,3 +1,38 @@
 [tool.black]
 line-length = 110
 include = '(\.py|^/test/check-.*|^/test/files/mock-insights)$'
+
+[tool.mypy]
+scripts_are_modules = true  # allow checking all scripts in one invocation
+explicit_package_bases = true
+# TODO: remove this when adding mypy config from other cockpit projects
+ignore_missing_imports = true
+
+[tool.pyright]
+strict = ["**"]
+extraPaths = ["test/common", "bots"]
+
+[tool.ruff]
+exclude = [
+    ".git/",
+    "modules/",
+    "node_modules/",
+]
+line-length = 118
+preview = true
+src = []
+
+[tool.ruff.lint]
+# TODO: fix ruff config when fixing python static checks
+ignore = [
+    "F403",
+    "F405",
+]
+
+[tool.ruff.lint.flake8-pytest-style]
+fixture-parentheses = false
+mark-parentheses = false
```

Comment:
This is only activated when we add `"PT"` rules and since this project doesn't use pytest I think we can skip it.